diff --git a/node_modules/expo-constants/scripts/get-app-config-android.gradle b/node_modules/expo-constants/scripts/get-app-config-android.gradle
index 6dc16fc..c8f4a21 100644
--- a/node_modules/expo-constants/scripts/get-app-config-android.gradle
+++ b/node_modules/expo-constants/scripts/get-app-config-android.gradle
@@ -2,14 +2,16 @@
 
 import org.apache.tools.ant.taskdefs.condition.Os
 
+apply from: new File(rootProject.projectDir, 'node-path.gradle')
+def awalletNodeExecutable = gradle.ext.awalletNodeExecutable
 
 def expoConstantsDir = project.providers.exec {
   workingDir(projectDir)
-  commandLine("node", "-e", "console.log(require('path').dirname(require.resolve('expo-constants/package.json')));")
+  commandLine(awalletNodeExecutable, "-e", "console.log(require('path').dirname(require.resolve('expo-constants/package.json')));")
 }.standardOutput.asText.get().trim()
 
 def config = project.hasProperty("react") ? project.react : [];
-def nodeExecutableAndArgs = config.nodeExecutableAndArgs ?: ["node"]
+def nodeExecutableAndArgs = config.nodeExecutableAndArgs ?: [awalletNodeExecutable]
 
 afterEvaluate {
   def projectRoot = file("${rootProject.projectDir}")
