diff --git a/node_modules/react-native-worklets/android/build.gradle b/node_modules/react-native-worklets/android/build.gradle
index 021f5bc..dc76816 100644
--- a/node_modules/react-native-worklets/android/build.gradle
+++ b/node_modules/react-native-worklets/android/build.gradle
@@ -12,6 +12,8 @@ def safeAppExtGet(prop, fallback) {
     appProject?.ext?.has(prop) ? appProject.ext.get(prop) : fallback
 }
 
+def awalletNodeExecutable = safeExtGet('awalletNodeExecutable', new File(rootProject.projectDir, 'tools/node').absolutePath)
+
 def isNewArchitectureEnabled() {
     // To opt-in for the New Architecture, you can either:
     // - Set `newArchEnabled` to true inside the `gradle.properties` file
@@ -30,7 +32,7 @@ def resolveReactNativeDirectory() {
     def reactNativePackage = file(
         providers.exec {
             workingDir(rootDir)
-            commandLine("node", "--print", "require.resolve('react-native/package.json')")
+            commandLine(awalletNodeExecutable, "--print", "require.resolve('react-native/package.json')")
         }.standardOutput.asText.get().trim()
     )
     if (reactNativePackage.exists()) {
@@ -294,7 +296,7 @@ android {
 
 def validateReactNativeVersionResult = providers.exec {
     workingDir(projectDir.path)
-    commandLine("node", "./../scripts/validate-react-native-version.js", REACT_NATIVE_VERSION.toString())
+    commandLine(awalletNodeExecutable, "./../scripts/validate-react-native-version.js", REACT_NATIVE_VERSION.toString())
     ignoreExitValue = true
 }
 
